src/test/kotlin/it/pagopa/aca/service/AcaServiceTests.kt

```diff
@@ -151,7 +152,8 @@ class AcaServiceTests {
         val responseCreate = AcaTestUtils.debitPositionModelResponse(iupd)
         /* preconditions */
         given(gpdClient.getDebtPosition(any(), any())).willReturn(Mono.just(responseGetPosition))
-        given(ibansClient.getIban(any(), any())).willReturn(Mono.just(Pair(ibanTest, companyName)))
+        given(ibansClient.getIban(any(), any()))
+            .willReturn(Mono.just(Pair(ibanTestUpdate, companyName)))
         given(gpdClient.updateDebtPosition(any(), any(), any()))
```

Comment:
Use ArgumentCapture for check the value of any() contains iban updated 🙂
```suggestion
        given(gpdClient.updateDebtPosition(any(), any(), debitPositionToUpdateCapture.capture().))
        assertEquals(ibanUpdated, debitPositionToUpdateCapture.getValue().getIban())
```